renderdoc/os/posix/linux/linux_process.cpp

```diff
@@ -35,6 +38,42 @@ char **GetCurrentEnvironment()
   return environ;
 }
 
+std::vector<int> getSockets(pid_t childPid)
+{
+  std::vector<int> sockets;
+  string dirPath = StringFormat::Fmt("/proc/%d/fd", (int)childPid);
+  DIR *pDir = opendir(dirPath.c_str());
+  if(pDir == NULL)
+  {
+    RDCWARN("Cannot open directory %s", dirPath);
+    return sockets;
+  }
+  struct dirent *pDirent;
+  while((pDirent = readdir(pDir)) != NULL)
+  {
+    if(pDirent->d_type != DT_LNK)
+      continue;
+
+    std::stringstream target;
+    target << dirPath.c_str() << "/" << pDirent->d_name;
```

Comment:
Please use the existing string formatting `StringFormat::Fmt` rather than C++ string streams.